### ISSUE TITLE
SplitButton - Cannot go through options with NVDA

### ISSUE THREAD

**Issue by gusch** [Sage/carbon#6245]
### Current behaviour

With NVDA active, focus on a SplitButton.

I can't open the options list with down arrow.

When the list is opened, i can't navigate through the options with up/down arrows

### Expected behaviour

With NVDA active, if i focus on a SplitButton, I can open the options list with enter or down arrow and then navigate through the options with up/down arrows

### CodeSandbox or Storybook URL

https://carbon.sage.com/?path=/docs/split-button--default

### Carbon Version

119.3.0

### Design Tokens Version

4.17.0

### What browsers are you seeing the problem on?

Chrome

### What Operating System are you seeing the problem on?

Windows

### Confidentiality

- [X] I confirm there is no confidential or commercially sensitive information included.

**Comment by Parsium** [comment]
FE-6178

**Comment by DipperTheDan** [comment]
Good day @gusch,  

I have investigated this issue with our `MultiActionButton` and `SplitButton`. I believe that our components are behaving this way with NVDA due to the differences in NVDA’s `browse` and `focus` modes. [In this article](https://www.accessibility-developer-guide.com/knowledge/screen-readers/desktop/browse-focus-modes/) on the Accessibility Developer Guide, it states that “for elements allowing only basic interaction (for example links, buttons, and checkboxes), focus mode is not activated. Basic interaction means that only the Enter or Space key triggers some interaction”. When using a combobox (Select components in Carbon’s world) then this applies “If the element allows complex interaction (for example textareas or comboboxes), focus mode is activated”. To get around this and manually activate the `focus` mode, the user can press the NVDA key (which in my case is the `ctrl` key) and Space. When `ctrl + space` are used, the component’s (`SplitButton` or `MultiActionButton`) menu can be opened and navigated as expected using the keyboard. You’ll also notice an audible `beep` when doing this that basically states `focus mode activated`. When you then close the menu (by `Esc` key or clicking away) you then get a different audible beep to basically indicate `focus mode deactivated and you’re back in browse mode now`.

The example in [W3C -> menubutton](https://www.w3.org/WAI/ARIA/apg/patterns/menu-button/examples/menu-button-actions/) also behaves the same way as our own MultiActionButton and SplitButton. Press the `Down Arrow` and the menu button does not open but if you press `ctrl + space` it does.

If you are able to test this on your application and get back to me with the results of if this now behaves as expected, that would be greatly appreciated. 

Thank you.

**Comment by gusch** [comment]
Hello @DipperTheDan.
It is indeed working with focus mode activated.
So I will close the ticket.
Thanks.
